backend/kaos_backend/util/validators.py

```diff
@@ -45,6 +49,15 @@ def validate_model_directory(cls, dirs):
         if "model" not in dirs:
             raise InvalidBundleError("Missing model directory in source-code bundle")
 
+    @classmethod
+    def validate_train_file_is_executable(cls, train_file):
+        f = open(train_file)
+        first_line = f.readline()
+        if cls.SHEBANG not in first_line:
+            raise InvalidBundleError("The train file cannot be executed. \n"
+                  "Please add the line '#!/usr/bin/xenv python3' "
```

Comment:
more generic please

Reply:
This has been extended for serve as well